init/kibana.conf

```diff
@@ -1,4 +1,15 @@
+# Kibana 4
+#
+description     "Kibana 4"
+
+start on started elasticsearch
+stop on (stopped probelogger or runlevel [!345])
```

Comment:
as with the other PRs I'm not sure about the shutdown order. Why not shutdown once ProbeManager is stopped?